recipes/recipes_emscripten/pango/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ex
+
+# get meson to find pkg-config when cross compiling
+export PKG_CONFIG=$BUILD_PREFIX/bin/pkg-config
+export PKG_CONFIG_PATH_FOR_BUILD=$BUILD_PREFIX/lib/pkgconfig
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$BUILD_PREFIX/lib/pkgconfig
+
+meson_config_args=(
+    -Dintrospection=disabled # requires gobject-introspection as run-time dep
+    -Dfontconfig=enabled
+    -Dfreetype=enabled
+    -Dcairo=disabled
```

Comment:
Add TODO, try enabling cairo through a subsequent PR (some error comes up with this enabled, needs to be debugged)

Reply:
Done